src/Behat/Symfony2Extension/Context/Argument/ServiceArgumentResolver.php

```diff
@@ -74,9 +74,13 @@ private function resolveArgument($argument)
             return $service;
         }
 
-        $withParameters = $this->replaceParameters($container, $argument);
+        $resolvedParam = $this->replaceParameters($container, $argument);
 
-        return $this->escape($withParameters);
+        if (!is_string($resolvedParam)) {
```

Comment:
Preferences are fine, I have no real preference either way; it was just the smallest issue on this PR at the time. My preference is that it's more readable to have a conditional block rather than a ternary in this case, equally I won't argue the point as I don't have any strong concerns either way.

Now the PR is complete, however, I don't mind changing it if that's the style for this repo.